test/general/key.js

```diff
@@ -2224,6 +2257,15 @@ describe('Key', function() {
     expect(await user.verifyCertificate(pubKey.primaryKey, user.otherCertifications[0], [certifyingKey])).to.equal(openpgp.enums.keyStatus.revoked);
   });
 
+  it('Verify certificate of key with future creation date', async function() {
+    const { keys: [pubKey] } = await openpgp.key.readArmored(key_created_2030);
+    const user = pubKey.users[0];
+    expect(await user.verifyCertificate(pubKey.primaryKey, user.selfCertifications[0], [pubKey], pubKey.primaryKey.created)).to.equal(openpgp.enums.keyStatus.valid);
+    const verifyAllResult = await user.verifyAllCertifications(pubKey.primaryKey, [pubKey], pubKey.primaryKey.created);
+    expect(verifyAllResult[0]).to.have.property('valid', true);
```

Comment:
```suggestion
    expect(verifyAllResult[0].valid).to.be.true;
```
Just for consistency with the other tests